source/Cute/Commands/Type/TypeScaffoldCommand.cs

```diff
@@ -1,17 +1,97 @@
-﻿using Cute.Commands.BaseCommands;
+﻿using Contentful.Core.Models;
+using Cute.Commands.BaseCommands;
 using Cute.Commands.Login;
 using Cute.Config;
 using Cute.Lib.Contentful;
+using Cute.Lib.Enums;
+using Cute.Lib.Exceptions;
+using Cute.Lib.TypeGenAdapter;
 using Cute.Services;
+using Spectre.Console;
 using Spectre.Console.Cli;
+using System.ComponentModel;
 
 namespace Cute.Commands.Type;
 
-public class TypeScaffoldCommand(IConsoleWriter console, ILogger logger, ContentfulConnection contentfulConnection,
-    AppSettings appSettings) : BaseLoggedInCommand<LoggedInSettings>(console, logger, contentfulConnection, appSettings)
+public class TypeScaffoldCommand(IConsoleWriter console, ILogger<TypeScaffoldCommand> logger, ContentfulConnection contentfulConnection,
+    AppSettings appSettings, HttpClient httpClient) : BaseLoggedInCommand<TypeScaffoldCommand.Settings>(console, logger, contentfulConnection, appSettings)
 {
-    public override Task<int> ExecuteCommandAsync(CommandContext context, LoggedInSettings settings)
+    private readonly HttpClient _httpClient = httpClient;
+
+    public class Settings : LoggedInSettings
+    {
+        [CommandOption("-c|--content-type")]
+        [Description("Specifies the content type to generate types for. Default is all.")]
+        public string? ContentTypeId { get; set; } = null!;
+
+        [CommandOption("-o|--output")]
+        [Description("The local path to output the generated types to.")]
+        public string OutputPath { get; set; } = default!;
+
+        [CommandOption("-l|--language")]
+        [Description("The language to generate types for (TypeScript/CSharp/Excel).")]
+        public GenTypeLanguage Language { get; set; } = GenTypeLanguage.TypeScript!;
+
+        [CommandOption("-n|--namespace")]
+        [Description("The optional namespace for the generated type.")]
+        public string? Namespace { get; set; } = default!;
```

Comment:
We need aparameter to suppress warnings and just return true. Otherwise this can't easily be added to dev workflows. maybe a global --force switch?